include/nuttx/note/note_driver.h

```diff
@@ -35,6 +36,56 @@
  * Public Types
  ****************************************************************************/
 
+struct note_driver_s;
+
+struct note_driver_ops_s
```

Comment:
let's keep the callback has the similar argument and number as the global functions

Reply:
done